multipathd/multipathd.service.in

```diff
@@ -6,6 +6,8 @@ Wants=systemd-udevd-kernel.socket @MODPROBE_UNIT@
 After=systemd-udevd-kernel.socket @MODPROBE_UNIT@
 After=multipathd.socket systemd-remount-fs.service
 Before=initrd-cleanup.service
+Wants=multipathd-shutdown-if-idle.timer
```

Comment:
We shouldn't do this. The timer should be independently enabled by either the admin or the distro.

Reply:
ACK.  I don't remember why I put that there, but I'll remove it.